addon.json

```diff
@@ -0,0 +1,74 @@
+{
+    "identifier": "com.seeq.addons.udf_ui",
+    "name": "User Defined Functions Formula Editor",
+    "description": "A Seeq add-on tool for managing user-defined formula functions.",
+    "version": "1.0.0",
+    "license": "Seeq",
+    "icon": "fa fa-edit",
+    "tags": {
+        "maintainer": "Seeq"
+    },
```

Comment:
Maintainer was moved out of tags, so
```
    "tags": {
        "maintainer": "Seeq"
    },
```
should just be
```
    "maintainer": "Seeq",
```

Reply:
fixed in the next push!